### PR TITLE
Issue 88 mChoiceCast

### PR DESCRIPTION
**Pull request by spgarbet**
I got this working using recastRecords.

### CODE REVIEW

**Review by nutterb — APPROVED** [review]
* I came across a few failing tests. All but one of them required the mChoice fields, so I added the mChoiceCast after exporting the records and they passed.
* The one other test was failing because mChoiceCast assumed that all of the checkbox fields were in Records (it wasn't behaving when operating on a subset). I altered how checkbox field names are identified within the data to accommodate subsets. 

With these changes, I have all tests passing, and check runs cleanly. 

I'll let you have a chance to look at https://github.com/vubiostat/redcapAPI/pull/101/files#diff-af13045d8c37d922f1d9adea824ad8dc64a80dd1d41874b1bbd198807f21355bR41-R52 before merging.

**Comment by spgarbet** [comment]
Your changes make sense.

**Pull request merged**
